### PR TITLE
Fixed all broken links in landing README

### PR DESCRIPTION
**Pull request by blrhc**
Required making new `class` markdown file, followed Martin's template but may need to be corrected...

### CODE REVIEW

File: classes/general/featured-content.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+## Schemas
+
+|Version and Link                                           |   Testnet(s)     |Active|
+|:---------------------------------------------------------:|------------------|:----:|
+| [v0](../../schemas/general/featuredContent0.json)         | `Rome`           | `no` |
+
+## Properties
+### Rome
+##### In Class index 0
```
Comment (bwhm) [inline classes/general/featured-content.md:43]: The idea here is to show the `properties` by their `in_class_index`, so these should count from 0-2.

Do you mind fixing that?

File: classes/general/featured-content.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      "name": "Featured Albums",
+      "description": "Music albums featured in the Music tab.",
+      "type": "InternalVec",
+      "maxItems": 6,
+      "classId": "Music Album"
+    }
+
+```
+
+## Entities
```
Comment (bwhm) [inline classes/general/featured-content.md]: Not all classes will have "fixed" entities, so you can delete this here...

**Review by bwhm — CHANGES_REQUESTED** [review]
When/if you have time, feel free to update the `video` and `music` class.md docs as well :)

**Comment by blrhc** [comment]
> When/if you have time, feel free to update the `video` and `music` class.md docs as well :)

Will do this ASAP.

**Review by bwhm — APPROVED** [review]

**Pull request merged**